code/reposadolib/reposadocommon.py

```diff
@@ -230,6 +230,24 @@ def print_stderr(msg, *args):
         print >> sys.stderr, concat_message(msg, *args)
 
 
+def humanReadable(size_in_bytes):
+    """Returns sizes in human-readable units."""
+    try:
+        size_in_bytes = int(size_in_bytes)
+    except ValueError:
+        size_in_bytes = 0
+    units = [(" bytes", 2**10, int),
+             (" KiB", 2**20, float),
+             (" MiB", 2**30, float),
+             (" GiB", 2**40, float),
+             (" TiB", 2**50, float)]
+    for suffix, limit, cast_unit in units:
+        if size_in_bytes > limit:
+            continue
+        else:
+            return str(cast_unit(round(size_in_bytes/float(limit/2**10), 1))) + suffix
+
+
```

Comment:
I think if you feel strongly about "KiB/MiB/etc", your might consider ditching the 'HumanReadableSizes' preference and just always use human-readable sizes, since the output format is going to change either way.

In any case, if the output format is going to change, it would be best to bring the topic up for discussion on the reposado mailing list.